integration_tests/e2e/achieve-goal.cy.ts

```diff
@@ -71,6 +72,20 @@ describe('Achieve goal', () => {
       cy.get('.govuk-details__text').should('contain', 'Some optional text in the achievement note field')
     })
 
+    it('Confirm errors are displayed with optional note of more than 4000 characters', () => {
```

Comment:
For this (and the other tests which exercise the error message on length of field) can we also validate that the relevant field contains the repopulated value please?